frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboard.scss

```diff
@@ -0,0 +1,7 @@
+.odc-monitoring-dashboard-dropdown-options {
+    display: flex;
+    float: right;
+    align-self: flex-end;
+    margin-top: -65px;
```

Comment:
For now this is fine.
We should push back on UX for this in future because:
* we don't have components for this right now
* it's a contextual item that appears at the same height as the tabs that set the context below
* there's no consideration for responsive mobile layout. Right now it overlaps the tabs on mobile.

cc @openshift/team-devconsole-ux